spec/was_seed_preassembly/lib/thumbnail_generator_service_spec.rb

```diff
@@ -21,20 +16,12 @@
     end
 
     it 'generates jp2 from jpeg thumbnail and pushes to druid_tree content directory', :image_prerequisite do
-      allow(Dor::WASSeed::ThumbnailGeneratorService).to receive(:capture).and_return('')
+      allow(Dor::WASSeed::ThumbnailGeneratorService).to receive(:capture)
```

Comment:
rando question - this is a step that will go away when we no longer have a thumbnail-service, amirite?

Reply:
The ThumbnailGeneratorService is not at all related to https://github.com/sul-dlss/was-thumbnail-service/

Reply:
ah - i am confusing it with the "update-thumbnail-generator" step for was-seed-dissemination, which presumably will go away when was-thumbnail-server goes away.  Imagine my getting those confused.  🙄

Reply:
Yes, it is very confusing.